src/test/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/RunPackageStepIT.groovy

```diff
@@ -167,6 +167,57 @@ class RunPackageStepIT extends IntegrationTestBase {
             jenkins.assertLogContains("Executing package 'test.pkg'", run)
     }
 
+    def 'Run pipeline by declaring .tbc and .tcf files in testConfig'() {
+        given:
+            WorkflowJob job = jenkins.createProject(WorkflowJob.class, 'pipeline')
+            job.setDefinition(new CpsFlowDefinition("node { ttRunPackage testCasePath: 'test.pkg', " +
+                    "testConfig: [tbcPath: 'test.tbc', tcfPath: 'test.tcf'] }", true))
+
+            GroovyMock(RestApiClientFactory, global: true)
+            RestApiClientFactory.getRestApiClient() >> new MockRestApiClient()
+
+        expect:
+            WorkflowRun run = job.scheduleBuild2(0).get()
+            jenkins.assertLogContains("Executing package 'test.pkg'...", run)
+            jenkins.assertLogContains("-> With TBC=test.tbc", run)
+            jenkins.assertLogContains("-> With TCF=test.tcf", run)
+    }
+
+    def 'Run pipeline by declaring KEEP in testConfig'() {
+        given:
+            WorkflowJob job = jenkins.createProject(WorkflowJob.class, 'pipeline')
+            job.setDefinition(new CpsFlowDefinition("node { ttRunPackage testCasePath: 'test.pkg', " +
+                    "testConfig: [tbcPath: 'KEEP', tcfPath: 'KEEP'] }", true))
+
+            GroovyMock(RestApiClientFactory, global: true)
+            RestApiClientFactory.getRestApiClient() >> new MockRestApiClient()
+
+        expect:
+            WorkflowRun run = job.scheduleBuild2(0).get()
```

Comment:
Do you test the behavior of KEEP with a real ecu.test running a package with the loaded config?

Reply:
removed `KEEP` as discussed